ui/src/shared/components/dateRangePicker/DatePicker.tsx

```diff
@@ -1,35 +1,56 @@
 // Libraries
-import React, {PureComponent} from 'react'
+import React, {PureComponent, ChangeEvent} from 'react'
 import ReactDatePicker from 'react-datepicker'
+import moment from 'moment'
 
 // Components
-import {Form, Input, Grid} from '@influxdata/clockface'
+import {Input, Grid, ComponentStatus} from '@influxdata/clockface'
 
 // Styles
 import 'react-datepicker/dist/react-datepicker.css'
 
 // Types
 import {Columns, ComponentSize} from '@influxdata/clockface'
+import {Form} from 'src/clockface'
```

Comment:
Since we used the workaround of passing in the unicode space characters when there is no error, do we still need to use the local `Form`? My hunch is we could import the clockface version and this should still work

Reply:
Ah, good point, thanks!